libcloud/compute/drivers/ecs.py

```diff
@@ -1289,6 +1289,102 @@ def create_public_ip(self, instance_id):
             return findtext(resp.object, 'IpAddress',
                             namespace=self.namespace)
 
+    def ex_security_group_ingress_permission(self, group_id,
```

Comment:
If possible, it would be great if you could make this method look more like the methods in the EC2 and ClouStack driver (same method name and signature if possible) - https://github.com/apache/libcloud/blob/trunk/libcloud/compute/drivers/ec2.py#L4052